build/common.props

```diff
@@ -0,0 +1,23 @@
+<Project>
+  <Import Project="..\version.props" />
```

Comment:
Building this repo? It seems to work for me

![image](https://cloud.githubusercontent.com/assets/2696087/22560800/b6d59d7a-e92a-11e6-897f-58f75a20dbe9.png)